src/guide/blockchain/expressions.md

```diff
@@ -0,0 +1,17 @@
+# Expressions, Conditionals, Logic
```

Comment:
Outdated section. This was replaced by Custom Instructions, which should be documented with the Executor probably.